src/tilesource.js

```diff
@@ -191,7 +196,11 @@ $.TileSource = function( width, height, tileSize, tileOverlap, minLevel, maxLeve
         this.aspectRatio = ( options.width && options.height ) ?
             (  options.width / options.height ) : 1;
         this.dimensions  = new $.Point( options.width, options.height );
-        this.tileSize    = options.tileSize ? options.tileSize : 0;
+        
+        this.tileSize = options.tileSize ? options.tileSize : 0;
```

Comment:
Btw, of course you should remove the `this.tileSize` reference a few lines above this section and anywhere else in the file.